internal/npm_package/npm_package.bzl

```diff
@@ -91,3 +99,18 @@ npm_package = rule(
     attrs = NPM_PACKAGE_ATTRS,
     outputs = NPM_PACKAGE_OUTPUTS,
 )
+"""
+The npm_package rule creates a directory containing a publishable npm artifact.
+
+Load it with
+`load("@build_bazel_rules_nodejs//:defs.bzl", "npm_package")`
+
+The default output is a directory containing the package contents.
+
+It also produces two named outputs, `[name].pack` and `[name].publish`. These can be used with `bazel run`.
+
+For an `npm_package` rule named `my_package`, you can use
+
+- `bazel run :my_package:pack` to create a .tgz of the package which will be written to the current working directory
+- `bazel run :my_package.publish` to publish the package to the npm registry
```

Comment:
Is there a way to forward flags to the `npm` command? E.g. `--tag=next` needs to be used regularly